common/v2/utils/convert.spec.ts

```diff
@@ -51,3 +52,54 @@ describe('it converts balance to fiat', () => {
     expect(converted).toEqual(expected);
   });
 });
+
+describe('it Remove / Add commission from amount', () => {
+  it('remove commission from decimal amount', () => {
+    const expected = 195;
+    const amount = 200;
+    const converted = withCommission({
+      amount,
+      rate: MYC_DEXAG_COMMISSION_RATE,
+      substract: true
+    });
```

Comment:
These tests will break when the RATE changes.
Now that the utility accepts a dynamic rate, we can use it to calculate the expected value and make our tests more robust.  Maybe with another math library in devDeps?